tests/cypress/integration/canvas3d_functionality/case_56_canvas3d_functionality_basic_actions.js

```diff
@@ -0,0 +1,158 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const_canvas3d';
+// Firefox does not yet support WebGL in headless mode: https://bugzilla.mozilla.org/show_bug.cgi?id=1375585 (disabled in the cypress_cron_type.json)
+context('Canvas 3D functionality. Basic actions.', () => {
+    const caseId = '56';
+    const screenshotsPath =
+        'cypress/screenshots/canvas3d_functionality/case_56_canvas3d_functionality_basic_actions.js';
+
+    function compareImages(imgBefore, imgAfter) {
+        cy.compareImages(`${screenshotsPath}/${imgBefore}`, `${screenshotsPath}/${imgAfter}`).then((diffPercent) => {
+            expect(diffPercent).to.be.gt(0);
+        });
+    }
+
+    function checkFilenameWrapperText(expectedText) {
+        cy.get('.cvat-player-filename-wrapper').should('contain.text', expectedText);
+    }
+
+    function pressKeyPerspectiveDirections(key, screenshotNameBefore, screenshotNameAfter) {
```

Comment:
The function name does not correspond to the things the function actually does.
I would suggest renaming it to ``testPerspectiveChangeOnKeyPress`` for example.

Reply:
Renamed